helm/prometheus-rules/templates/platform/atlas/alerting-rules/loki.rules.yml

```diff
@@ -83,3 +83,28 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    - alert: LokiNeedsToBeScaledDown
+      annotations:
+        description: 'Loki is consuming very few resources and needs to be scaled down.'
+        opsrecipe: loki/
+      expr: |-
+        sum by (cluster_id, installation, namespace, pipeline, provider) (container_memory_working_set_bytes{container="loki", namespace="loki"}) 
+          / 
+        sum by(cluster_id, installation, namespace, pipeline, provider) (kube_pod_container_resource_requests{container="loki", namespace="loki", unit="byte"}) 
+          <= 0.30 
+        and
+        sum(rate(container_cpu_usage_seconds_total{container="loki", namespace="loki"}[5m])) by (cluster_id, installation, namespace, pipeline, provider) 
+          / 
+        sum by(cluster_id, installation, namespace, pipeline, provider) (kube_pod_container_resource_requests{container="loki", namespace="loki", unit="core"}) 
+          <= 0.30
+      for: 60m
```

Comment:
60m is too short IMO, I would prefer that we monitor this over a longer period of time like 1d or even more if possible. Resource usage might fluctuate in different periods of time like weekend, holidays and so on.

Reply:
True, I think 1d is fine :)

Reply:
Updated it to 1d.